README.md

```diff
@@ -113,19 +133,25 @@ cd ..
 
 The ZIP file must have `index.js` at the root. That’s why you must run `zip` from within the `build` directory.
 
-Now we create a function with the name `wikipedia`:
+Next, put the package in the bucket you previously created.
+
+```console
+npx stedi buckets put-object --key package.zip --body file://build/package.zip --bucket-name <YOUR_BUCKET_NAME_HERE>
+```
+
+Now we create a function with the name `wikipedia`.
 
 ```console
-npx stedi functions create-function --function-name wikipedia --package build/package.zip
+npx stedi functions create-function --function-name wikipedia --package "s3://<YOUR_BUCKET_NAME_HERE>/package.zip"
```

Comment:
Should we update the CLI to prepend the `s3://` prefix automatically? I'd prefer that (maybe only if it isn't already added by the customer). It would be more consistent with the AS2 send-file CLI that way, and we're not unnecessarily leaking the S3 detail.

Reply:
I think we should update the CLI to use a working default bucket in case not specified (similar to how terminal does this). it will improve the experience of getting started significantly

Reply:
but in a different PR/release of the CLI - one thing at a time